pkg/reporter/json_report.go

```diff
@@ -215,6 +215,9 @@ func (r *jsonReportGenerator) buildSpecReport() (*schema.Report, error) {
 }
 
 func (j *jsonReportGenerator) buildJsonPackageReportFromPackage(p *models.Package) *jsonreportspec.PackageReport {
+
+	flag := true
```

Comment:
Can you describe the purpose of `flag` here? I do not see it being updated anywhere.